src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/ApplicationGatewayTests.ps1

```diff
@@ -24,15 +24,17 @@ function Test-ApplicationGatewayCRUD
 	$resourceTypeParent = "Microsoft.Network/applicationgateways"
 	$location = Get-ProviderLocation $resourceTypeParent
 
-	$rgname = "rg01"
```

Comment:
instead of hardcoding variable names,
use this instead 
 $rgname = Get-ResourceGroupName
$vnetName = Get-ResourceName